__tests__/integration.js

```diff
@@ -0,0 +1,391 @@
+const awsServerlessExpress = require('../index')
+const apiGatewayEvent = require('../example/api-gateway-event.json')
+const app = require('../example/app')
+
+const server = awsServerlessExpress.createServer(app)
+const lambdaFunction = {
```

Comment:
This is our mock Lambda function handler. See https://github.com/awslabs/aws-serverless-express/blob/master/example/lambda.js and you'll see this is the core of our Lambda function. This server/"lambda function" will be used in the majority of our tests. For tests with special cases (e.g. base64 encoding images; server callback; socket path conflict error/edge case) a new server is created with special configuration.